src/components/Modals/CreateGnosisSafeModal/styles.scss

```diff
@@ -34,4 +39,21 @@
   > label {
     margin: 15px 0 5px 0;
   }
+
+  .modal-footer {
+    margin-top: 15px;
+    border-top: 1px solid #CBCBCC;
+    padding: 10px 0;
+
+    & > * + * {
```

Comment:
Just "taste" - but for me, it looks not readable "at first glance". I would suggest rewriting it to something more explicit.